addons/web/static/src/scss/bootstrap_review.scss

```diff
@@ -39,6 +39,9 @@
         // TODO remove me in master.
         background-color: transparent !important;
     }
+    table {
+        color: inherit;
+    }
```

Comment:
your commit mentions the portal module but your changes are made in web and website

Reply:
Actually the issue is not specific to portal.
If you add a table with the text editor in a card from the columns snippet you will have the same issue.